### PR TITLE
style: fix the heading size

### PR DESCRIPTION
**Pull request by anupriya0703**
This PR fixes the heading size of sub-headers under [Control Plane.](https://mayastor.gitbook.io/introduction/quickstart/deploy-mayastor#core-agents)

Resolves: https://github.com/openebs/mayastor-docs/issues/67

Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: quickstart/deploy-mayastor.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ mayastor-csi   3         3         3       3            3           kubernetes.i
 
 ### Control Plane
 
-### Core Agents
+## Core Agents
```
Comment (GlennBullingham) [inline quickstart/deploy-mayastor.md]: The indentation is the wrong way around:  The various control plane components e.g. "Core Agents" should be a sub-ordinate heading, under Control Plane.

i.e. 

## Control Plane
...
### Core Agents
...
### REST
etc.

File: quickstart/deploy-mayastor.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ mayastor-csi   3         3         3       3            3           kubernetes.i
 
 ### Control Plane
```
Comment (GlennBullingham) [inline quickstart/deploy-mayastor.md]:
```suggestion
## Control Plane
```

File: quickstart/deploy-mayastor.md (hunk 1 above)
Comment (GlennBullingham) [inline quickstart/deploy-mayastor.md]:
```suggestion
### Core Agents
```

File: quickstart/deploy-mayastor.md [hunk 3]
```diff
@@ -183,7 +183,7 @@ core-agents-5f4d9f786b-6vvxc   1/1     Running   0          117s
 {% endtab %}
 {% endtabs %}
 
-### REST
+## REST
```
Comment (GlennBullingham) [inline quickstart/deploy-mayastor.md]:
```suggestion
### REST
```

File: quickstart/deploy-mayastor.md [hunk 4]
```diff
@@ -218,7 +218,7 @@ rest-5cd9665499-cdgmm   1/1     Running   0          2m35s
 {% endtab %}
 {% endtabs %}
 
-### CSI Controller
+## CSI Controller
```
Comment (GlennBullingham) [inline quickstart/deploy-mayastor.md]:
```suggestion
### CSI Controller
```

File: quickstart/deploy-mayastor.md [hunk 5]
```diff
@@ -246,7 +246,7 @@ csi-controller-579f77f64-7dq7g   3/3     Running   0          39m
 {% endtab %}
 {% endtabs %}
 
-### MSP Operator
+## MSP Operator
```
Comment (GlennBullingham) [inline quickstart/deploy-mayastor.md]:
```suggestion
### MSP Operator
```

File: quickstart/deploy-mayastor.md (hunk 2 above)
Comment (GlennBullingham) [reply quickstart/deploy-mayastor.md]: Control Plane should be the dominant heading

**Review by GlennBullingham — CHANGES_REQUESTED** [review]
The indentation levels are the wrong way around: The various control plane components e.g. "Core Agents" should be sub-ordinate headings, under "Control Plane".

File: quickstart/deploy-mayastor.md (hunk 1 above)
Comment (anupriya0703) [reply quickstart/deploy-mayastor.md]: Oh! I mixed that up. Thanks for pointing out. Fixed.

**Review by avishnu — APPROVED** [review]
lgtm

**Review by GlennBullingham — APPROVED** [review]

**Pull request merged**